R/ResamplePrediction.R

```diff
@@ -16,8 +16,8 @@ NULL
 
 
 makeResamplePrediction = function(instance, preds.test, preds.train) {
-  tenull = sapply(preds.test, is.null)
-  trnull = sapply(preds.train, is.null)
+  tenull = vapply(preds.test, is.null, FUN.VALUE = logical(1))
+  trnull = vapply(preds.train, is.null, FUN.VALUE = logical(1))
   if (any(tenull)) pr.te = preds.test[!tenull] else pr.te = preds.test
   if (any(trnull)) pr.tr = preds.train[!trnull] else pr.tr = preds.train
```

Comment:
If we assume that `preds.test` and `preds.train` never contains 'NULL' (as we discussed in our weekly meeting), we could even remove lines 19 - 22. If not, I think we can at least prettify the code and replace lines 19 - 22 with `pr.te = filterNull(preds.test)` and `pr.tr = filterNull(preds.train)`.